includes/Modules/AdSense.php

```diff
@@ -738,6 +769,42 @@ private function register_tag() {
 		}
 	}
 
+	/**
+	 * Returns the decoded ad blocking recovery tag from the options.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return string|null The decoded ad blocking recovery tag, or null if not available.
+	 */
+	public function get_ad_blocking_recovery_tag() {
+		$encoded_tag = $this->options->get( self::AD_BLOCKING_RECOVERY_TAG_OPTION );
+		if ( ! $encoded_tag ) {
+			return null;
+		}
+
+		$decoded_tag = base64_decode( $encoded_tag, true );
+
+		return false === $decoded_tag ? null : $decoded_tag;
+	}
+
+	/**
+	 * Returns the decoded ad blocking recovery error protection tag from the options.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return string|null The decoded ad blocking recovery error protection tag, or null if not available.
+	 */
+	public function get_ad_blocking_recovery_error_protection_tag() {
+		$encoded_tag = $this->options->get( self::AD_BLOCKING_RECOVERY_ERROR_PROTECTION_TAG_OPTION );
+		if ( ! $encoded_tag ) {
+			return null;
+		}
+
+		$decoded_tag = base64_decode( $encoded_tag, true );
+
+		return false === $decoded_tag ? null : $decoded_tag;
+	}
```

Comment:
Let's extract a dedicated `Setting` for this instead of building it into the Module's class, then we could expose a getter for that instance, similar to `get_settings` to minimize the changes here.